fxa/requests.py

```diff
@@ -10,7 +10,10 @@
 from requests.auth import AuthBase
 from six.moves.urllib.parse import urlparse
 
-from fxa.core import DEFAULT_SERVER_URL, Client
+from fxa import core
+from fxa import oauth
+
+DEFAULT_CLIENT_ID = "5882386c6d801776"  # Firefox dev Client ID
```

Comment:
I don't think we should bake in a default clientID here, if you're using this for anything then you're going to need to be registered as an oauth client somehow.  If you feel stringly about it, I'm open to being talked into provisioning a separate "PyFxA development" client id.